### PR TITLE
Password Strength Checker in C++

### PR DESCRIPTION
**Pull request by japneetkaurbhatia**
### Your checklist for this pull request
:warning: Please review the [guidelines for contributing](https://github.com/sanscript-tech/hacking-tools-scripts/blob/main/CONTRIBUTING.md) to this repository.

Parent issue: #299

### Type of change

**Select all that applies**:
* [x] New feature (non-breaking change which adds functionality)
* [ ] Bug fix (non-breaking change which fixes an issue)
* [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
* [ ] This change requires a documentation update

### Checklist:

* [x] My code follows the style guidelines(Clean Code) of this project.
* [x] I have performed a self-review of my own code.
* [x] I have used consistent code conventions and clear function/method/variable names.
* [x] I have commented on my code, particularly in hard-to-understand areas.
* [x] I have created a helpful and easy to understand README.md which contains description, requirements and use case.
* [x] I have added relevent screenshots/images if necessary in a seperate folder.
* [x] I have added my work in respective language folder.
* [x] My changes generate no new warnings.

### CODE REVIEW

File: C++/PasswordStrength/README.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+Aim: To check strength of Password in C++
+
+If Password is 8-character long and have uppercase, lowercase , digit and special character then it is Strong
+
+If Password is 6-chacter long with uppercase or Lowercase and special character then it is Moderate
+
+otherwise its Weak
```
Comment (yashaswibiyahut) [inline C++/PasswordStrength/README.md:7]: Add sample use case or example

File: C++/PasswordStrength/checkpassword.cpp [hunk 2]
```diff
@@ -0,0 +1,36 @@
+#include <iostream>
+
```
Comment (yashaswibiyahut) [inline C++/PasswordStrength/checkpassword.cpp:2]: Remove all extra spaces

File: C++/PasswordStrength/checkpassword.cpp [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+using namespace std;
+
+void CheckPassword(string& input)
```
Comment (yashaswibiyahut) [inline C++/PasswordStrength/checkpassword.cpp:5]: Add proper comments

File: C++/PasswordStrength/README.md [hunk 4]
```diff
@@ -0,0 +1,7 @@
+Aim: To check strength of Password in C++
+
+If Password is 8-character long and have uppercase, lowercase , digit and special character then it is Strong
+
+If Password is 6-chacter long with uppercase or Lowercase and special character then it is Moderate
+
```
Comment (yashaswibiyahut) [inline C++/PasswordStrength/README.md:6]: Add time and space complexity

**Review by yashaswibiyahut — CHANGES_REQUESTED** [review]

**Comment by yashaswibiyahut** [comment]
Please squash the commits

**Comment by tejan-singh** [comment]
@japneetbhatia do not send a pull request from the main branch, Create another branch and send pr from that branch. Read contribution guidelines for more details.

**Comment by japneetkaurbhatia** [comment]
Ok @tejan-singh @yashaswibiyahut  I'll correct this by today

**Pull request closed without merging**